### DISCUSSION TITLE
imixs plugin installed into Eclipse IDE, but not shown

### DISCUSSION THREAD

**Discussion by hurakdulger** [imixs/imixs-workflow#800] [Q&A] [answered]
I have installed imixs BPMN Modeller as described at webpage (https://www.imixs.org/doc/modelling/install.html). 
Then, I have restarted IDE and created new BPMN Model by following File > New > Other > BPMN2 > BPMN2 Model.

But there is no Workflow profile under Profiles category and there is no imixs-Workflow category exists (https://www.imixs.org/doc/modelling/eclipse_workbench.html).

What I am doing wrong?

Thanx for advance.

**Comment by rsoika** [comment, marked as answer]
Hi,
what you need is to set the target namespace to:

    http://www.imixs.org/bpmn2


![image](https://user-images.githubusercontent.com/1339124/189345188-6a072dd8-9d89-442a-a813-bebbde7929f6.png)

You can also change this attribute later in your .bpmn file directly

**Comment by hurakdulger** [comment]
![image](https://user-images.githubusercontent.com/53187728/189349617-90c854a9-7cc4-4269-b215-b9b76c41594e.png)

It worked. Thank you @rsoika 

Regards.
